.github/workflows/main.yml

```diff
@@ -1124,9 +1102,9 @@ jobs:
           asset_name: libvcx-ios-${{ needs.workflow-setup.outputs.PUBLISH_VERSION }}-device.zip
           asset_content_type: application/zip
 
-  release-ios-universal:
+  publish-ios-universal:
```

Comment:
This name is confusing, as it uses the same word - "publish" - in a different sense than publishing docker image. I uploads *release assets*, which are used only in case a release is made.